std/math.d

```diff
@@ -3927,6 +3927,91 @@ float floor(float x) @trusted pure nothrow @nogc
     assert(isNaN(floor(float.init)));
 }
 
+/**
+ * Round `val` to a multiple of `unit`. `rfunc` specifies the rounding
+ * function to use; by default this is `rint`, which uses the current
+ * rounding mode.
+ */
+real quantize(alias rfunc = rint)(real val, real unit)
+    if (isCallable!rfunc)
+{
+    real ret = val;
+    if (unit != 0)
+    {
+        const scaled = val / unit;
```

Comment:
@JackStouffer Because it doesn't really make a difference here, and `const` is less typing. I'll change it if you insist, though.

(`immutable` can be meaningfully different than `const` when the value can be computed at compile time, or it lives on the heap. Neither of those conditions is met here, though; `scaled` must be computed at run time, and it should spend the entirety of its very brief lifetime in a register.)